content/applications/websites/ecommerce/products.rst

```diff
@@ -1,3 +1,9 @@
+:show-content:
+
+========
+Products
+========
+
 ==================
 Product management
 ==================
```

Comment:
```suggestion
Product management
==================
```
I agree with @auva-odoo ... this is how this header should be formatted in this case